_data/projects/search/releases/5.10/series.yml

```diff
@@ -30,7 +30,7 @@ artifacts:
     summary: WildFly feature pack - JGroups backend
 integration_constraints:
   java:
-    version: 8 - 9
+    version: 8 or 9
```

Comment:
9?

Reply:
Right, it wasn't really official support... and we switched to 10 after some time, and currently we don't test against JDK9.

I'll remove it.